# time when command is sent
std_msgs/Header header 

float32 record_duration_s 